components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -81,8 +81,8 @@ spec:
           env:
             - name: TYK_MDCB_LISTENPORT
               value: "{{ .Values.mdcb.listenPort }}"
-            - name: TYK_MDCB_HEALTHCHECKPORT
-              value: "{{ .Values.mdcb.probes.healthCheckPort }}"
+            - name: TYK_MDCB_HTTPPORT
+              value: "{{ .Values.mdcb.probes.httpPort }}"
```

Comment:
There are other references to `healthCheckPort` in the file. Can you update those too?
Also, you will need to update service-mdcb.yaml file too